python/src/opendp/mod.py

```diff
@@ -432,6 +436,17 @@ class Queryable(object):
     def __init__(self, value, query_type):
         self.value = value
         self.query_type = query_type
+        self.__opendp_dict__ = {
+            'func': 'Queryable',
+            'module': 'mod',
+            'kwargs': {
+                'value': value,
+                'query_type': query_type,
+            },
+        }
```

Comment:
In the future, some queryables may be serializable, but it would be universal. Worth adding a special case error message.

Reply:
special message added, though not sure it shows in every desired case.